src/less/actionable/ds4/actionable.less

```diff
@@ -1,20 +1,39 @@
 @import "../../variables/ds4/color-variables.less";
-@import "../../variables/ds4/icon-variables.less";
 @import "../../variables/ds4/typography-variables.less";
 
+// CSS Custom properties
+a.icon-link,
+button.icon-btn {
+    --actionable-badge-border-color: @color-background-default;
+    --actionable-icon-foreground-color: @color-icon-actionable-default;
+    --actionable-icon-foreground-color-active: @color-icon-actionable-default;
+    --actionable-icon-foreground-color-hover: @color-text-default;
+    --actionable-icon-foreground-color-visited: @color-text-default;
+    --actionable-image-border-color-hover: @color-core-gray-spanish;
+}
+
+a.img-link,
+button.img-btn {
+    --actionable-image-background-color-visited: @color-grey2;
+    --actionable-image-border-color-active: @color-text-secondary;
+    --actionable-image-border-color-hover: @color-grey5;
+}
+
+// IE11 fallbacks
+@actionable-badge-border-color: @color-background-default;
+@actionable-badge-font-size: @font-size-12;
 @actionable-icon-foreground-color: @color-icon-actionable-default;
 @actionable-icon-foreground-color-active: @color-icon-actionable-default;
-@actionable-icon-foreground-color-visited: @color-text-default;
 @actionable-icon-foreground-color-hover: @color-text-default;
-@actionable-badge-border-color: @color-background-default;
-@actionable-badge-font-size: @font-size-12;
+@actionable-icon-foreground-color-visited: @color-text-default;
+@actionable-image-background-color-visited: @color-core-gray-light;
 @actionable-image-border-color-hover: @color-core-gray-spanish;
 @actionable-image-border-color-active: @color-core-gray-dim;
-@actionable-image-background-color-visited: @color-core-gray-light;
 
 @import "../base/actionable.less";
 
 // todo: normalize ds4 & ds6
+/* stylelint-disable-next-line no-duplicate-selectors */
```

Comment:
I'm curious, why are we having duplicate selectors? Is this part of this PR or going to be like that for future PRs with variables

Reply:
I just wanted to keep the style block declaration for custom properties separate from the block for styling. I think in this specific case I'm going to go ahead and normalize this block by using a grey color instead of opacity for ds4.

Reply:
Done.